activesupport/lib/active_support/cache/redis_cache_store.rb

```diff
@@ -486,6 +470,36 @@ def serialize_entries(entries, **options)
           end
         end
 
+        def change_counter(key, amount, options)
+          redis.with do |c|
+            c = c.node_for(key) if c.is_a?(Redis::Distributed)
+
+            if options[:expires_in] && supports_expire_nx?
+              c.pipelined do |p|
```

Comment:
This shadows `Kernel#p` and is also a little cryptic. What about `c.pipelined do |pipe|` or `c.pipelined do |pipeline|`?